Add `Expert Global Taiko Showdown 2025 Concludes` news post (#13208)
ppy/osu-wiki: news/2025/2025-05-06-expert-global-taiko-showdown-2025-concludes.md

    </audio>
</div>

We then moved on to quite a different style for this tiebreaker, being a more rock/speedcore-focused song, rhythms changing all over the board. This was a song of professional grade, despite the composer, [trailmix37](https://soundcloud.com/trailmix38), being completely unknown with just 52 subscribers on SoundCloud and not a single osu! map submitted for any of his 4 past songs. That said, if you were to play this without knowing any of this context? You would not notice it whatsoever.

Mappers [davidminh0111](https://osu.ppy.sh/users/2140739) and [SolaEclipse](https://osu.ppy.sh/users/2140739) did not hold back after hearing this song, with some of these sections vying to trip you up no matter which mod you choose, and you can see this even in the matches where players attempted this map.

### Round of 16

[![](/wiki/shared/news/2025-05-06-expert-global-taiko-showdown-2025-concludes/banner-the-great-revelation-by-mother-brain.png)](https://osu.ppy.sh/beatmapsets/2329073)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/media/news/egts-2025-the-great-revelation-by-mother-brain.mp3">
    </audio>
</div>

> This one is just shooting to be a massive bomb. Welcome to 5 minutes of Frenchcore by [Mothtek](https://soundcloud.com/mothtek).

Reviewer: ```suggestion
This one is just shooting to be a massive bomb. Welcome to 5 minutes of frenchcore by [Mothtek](https://soundcloud.com/mothtek).
```

A massive departure from the melancholic hype that the two prior songs possessed. A song by one of the more prolific artists in the Japanese hardtek scene. A song named "[The Great Revelation By Mother Brain](https://soundcloud.com/mothtek/mothtek-the-great-revelation-by-mother-brainegts2025-round-of-16-tiebreaker)". When does it stop? The answer is never (figuratively).

Mappers [Ideal](https://osu.ppy.sh/users/3869519) and [Raphalge](https://osu.ppy.sh/users/3918650) tried to reel this insane song in a bit by toning down the mechanical difficulty compared to the Round of 32, as it instead aims to be played with mods by the players left, but even then, it is a struggle for sure.

### Quarterfinals

[![](/wiki/shared/news/2025-05-06-expert-global-taiko-showdown-2025-concludes/banner-atomic-disassembler.png)](https://osu.ppy.sh/beatmapsets/2332724)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/media/news/egts-2025-atomic-disassembler.mp3">
    </audio>
</div>

[Atomic Disassembler.](https://soundcloud.com/yusuke-7328/atomic-disassembler)

> This title summarises the goal behind this song pretty well. The tiebreaker for this year's Quarterfinals by [CS4W](https://soundcloud.com/yusuke-7328) is a Gothic Hardcore song mixed with Hi-tech and Drum & Bass elements, an insane mixes of genres to be sure.

Reviewer: ```suggestion
This title summarises the goal behind this song pretty well. The tiebreaker for this year's Quarterfinals by [CS4W](https://soundcloud.com/yusuke-7328) is a Gothic Hardcore song mixed with Hi-tech and Drum & Bass elements, an insane mix of genres to be sure.
```

The map by [My Angel Seia](https://osu.ppy.sh/users/17797595) has absolutely nothing angelic about it. The easy parts are when 220 BPM streams are mapped at the normal SV, but whenever you see a single SV slowdown in this map, the only answer is to run, frankly.

### Semifinals

[![](/wiki/shared/news/2025-05-06-expert-global-taiko-showdown-2025-concludes/banner-white-tiger.png)](https://osu.ppy.sh/beatmapsets/2336579#taiko/5016172)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/media/news/egts-2025-white-tiger.mp3">
    </audio>
</div>

> [Xyris](https://soundcloud.com/itsxyris), being no stranger to the osu! tournament scene, definitely understood the assignemnt here as well. He throws out crazy mixes of Speedcore and Dubstep to make for some of the most challenging bursts.

Reviewer: ```suggestion
[Xyris](https://soundcloud.com/itsxyris), being no stranger to the osu! tournament scene, definitely understood the assignment here as well. He throws out crazy mixes of speedcore and dubstep to make for some of the most challenging bursts.
```

### Grand Finals

[![](/wiki/shared/news/2025-05-06-expert-global-taiko-showdown-2025-concludes/banner-absolute-disorder.png)](https://osu.ppy.sh/beatmapsets/2343534)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/media/news/egts-2025-absolute-disorder.mp3">
    </audio>
</div>

This is the only non-tiebreaker song to be discussed here, but with good reason to do so.

Our Grand Finals HR2 pick, [Acute Disarray - AbsoluTe disoRdeR](https://soundcloud.com/abs_dsr/absolute-disorder), was an original song released for EGTS 2025, but also a 4-way collaboration with three of the rhythm games we've worked with throughout EGTS.

[Acute Disarray](https://soundcloud.com/abs_dsr), a unit of [AiSS](https://www.youtube.com/channel/UCySx3jmK0bjwiwix_EoKm4w) participating for [Orzmic](https://twitter.com/orzmic), [rN](https://soundcloud.com/rurinn) participating for [Phigros](https://twitter.com/Phigros_PGS), [takehirotei](https://soundcloud.com/takehirotei) participating for GTS and [Riya](https://soundcloud.com/riyaelectronic) participating for [A Dance of Fire and Ice](https://7thbe.at/), released the song on each respective game.

> The sheer scale this collaboration is on is definitely absurd, and the map made by [Hivie](https://osu.ppy.sh/users/14102976) for it definitely follows it through, ending up as a defining pick for both Grand Finals matches.

Reviewer: ```suggestion
The sheer scale of this collaboration is definitely absurd, and the map made by [Hivie](https://osu.ppy.sh/users/14102976) for it definitely follows it through, ending up as a defining pick for both Grand Finals matches.
```

Due to the special collaboration aspect of the song, we ask others not to map it, but to instead wait for a full spread on all game modes that we have planned. Do give it time.

---

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/S_00DzXfNNI" frameborder="0" allowfullscreen></iframe>
</div>

The final song of EGTS 2025, the Grand Finals tiebreaker, truly speaks for itself: [《wr/d.:colliders》\~Chaos in Conclusion\~](https://soundcloud.com/blobdash/wrld-colliders) by the GTS Sound Team.

> This song involved **20** composers, **35** mappers, and **15** designers to create **11 minutes** of song, beatmap *and* a full **music video** top top it off.

Reviewer: ```suggestion
This song involved **20** composers, **35** mappers, and **15** designers to create **11 minutes** of song, beatmap *and* a full **music video** to top it off.
```

Read on for a detailed rundown of the Grand Finals and [other highlights from this tournament](#match-highlights) below.

## Grand Finals recap

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/CVBCS3p2FZE" frameborder="0" allowfullscreen></iframe>
</div>

Those familiar with EGTS will know that the Grand Finals matchup always used to be fairly one-sided due to the skill disparity between top-seeded players:

- In 2020, ::{ flag=CA }:: [janitore](https://osu.ppy.sh/users/3307897) went up against ::{ flag=JP }:: [uone](https://osu.ppy.sh/users/5321719) who sought to flip the script after being knocked into the lower bracket in Semifinals. uone proceeded to sweep janitore into a bracket reset and finished the job by defeating him 7–2.
- In 2021, ::{ flag=JP }:: [ekumea1123](https://osu.ppy.sh/users/9119501) and ::{ flag=JP }:: [Seren58](https://osu.ppy.sh/users/15252950) gave us the closest Grand Finals scoreline thus far with Seren58 resetting the bracket with a 3–7 victory, followed by a 7–4.

> - In 2022, ::{ flag=JP }:: [Seren58](https://osu.ppy.sh/users/15252950) came back in 2022 to defeat ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) with a crushing 7–1 victory.

Reviewer: ```suggestion
- In 2020, ::{ flag=CA }:: [janitore](https://osu.ppy.sh/users/3307897) went up against ::{ flag=JP }:: [uone](https://osu.ppy.sh/users/5321719) who sought to flip the script after being knocked into the lower bracket in Semifinals. uone proceeded to sweep janitore into a bracket reset and finished the job by defeating him 7–2.
```

> - In 2022, ::{ flag=JP }:: [Seren58](https://osu.ppy.sh/users/15252950) came back in 2022 to defeat ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) with a crushing 7–1 victory.

Reviewer: ```suggestion
- In 2021, ::{ flag=JP }:: [ekumea1123](https://osu.ppy.sh/users/9119501) and ::{ flag=JP }:: [Seren58](https://osu.ppy.sh/users/15252950) gave us the closest Grand Finals scoreline thus far with Seren58 resetting the bracket with a 3-7 victory, followed by a 7–4.
```

> - In 2022, ::{ flag=JP }:: [Seren58](https://osu.ppy.sh/users/15252950) came back in 2022 to defeat ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) with a crushing 7–1 victory.

Reviewer: ```suggestion
- In 2022, ::{ flag=JP }:: [Seren58](https://osu.ppy.sh/users/15252950) came back in 2022 to defeat ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) with a crushing 7–1 victory.
```

> History seemed to repeat itself as ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) pulled ahead with a 5–2 scoreline against ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357), and most were expecting one of the best HD player of osu!taiko to win the next pick of HD2. Yet, miraculously, ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) found catastrophic misses in the final ten seconds which swayed the point, to the shock of everyone, to ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357)'s bitterly held FC.

Reviewer: ```suggestion
History seemed to repeat itself as ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) pulled ahead with a 5–2 scoreline against ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357), and most were expecting one of the best HD player of osu!taiko to win the next pick of HD2. Yet, miraculously, ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) found catastrophic misses in the final ten seconds which swayed the point, to the shock of everyone, to ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357)'s bitterly held FC.
```

> History seemed to repeat itself as ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) pulled ahead with a 5–2 scoreline against ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357), and most were expecting one of the best HD player of osu!taiko to win the next pick of HD2. Yet, miraculously, ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) found catastrophic misses in the final ten seconds which swayed the point, to the shock of everyone, to ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357)'s bitterly held FC.

Reviewer 2: subjective but sure

```suggestion
History seemed to repeat itself as ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) pulled ahead with a 5–2 scoreline against ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357), and most were expecting one of the best HD player in osu!taiko to win the next pick of HD2. Yet, miraculously, ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) found catastrophic misses in the final ten seconds which swayed the point, to the shock of everyone, to ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357)'s bitterly held FC.
```

::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073)'s final pick was between one of two incredibly painful maps: NM5, a chaotic polyrhythmic experience, and DT1, a speedy accuracy nightmare. His choice fell upon NM5, but no one could've anticipated the back-and-forth that ensued.

For the first and only time in the history of EGTS, players and viewers alike would be treated to a true world premiere: a genuine Grand Finals tiebreaker; an 11-minute gauntlet at the end of a nerve-wracking one-hour match to decide whether ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) would be crowned the final EGTS champion, or whether he would have to prove himself all over again. By the end, ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357) won with a 47,159-score difference.

Both players were given a desperately needed 10-minute break, as the bracket was reset.

---

The new match started yet again with a show of force in their own picks, yet the advantages didn't make for easy wins. Neither player willing to give up, both continued to flaunt their ambitions at the champion title.

> For example, ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) found a 1.042 *million* HD score on FM3 to win the point, yet it would be impossible to overstate ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357)'s FC with **HR** on his opponents reading- and accuracy-heavy pick.

Reviewer: ```suggestion
For example, ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) found a 1.042 *million* HD score on FM3 to win the point, yet it would be impossible to overstate ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357)'s FC with **HR** on his opponents reading and accuracy-heavy pick.
```

Both continued to not give away any of their own picks, but it was apparent that something had changed in ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073)'s mind. A 2x100 HD FC on FM1, a *halving* of his previous 100 miss count on DT2 (albeit still losing the pick), and his **near SS** on HD2 showed the audience just what kind of determination he had brought to the match that he wasn't willing to give up on.

<div align="center" class="osu-md__paragraph">
    <video width="95%" controls>
        <source src="https://assets.ppy.sh/media/news/egts-2025-grape_tea-vs-kanten_07.mp4" type="video/mp4" preload="none">
    </video>
</div>

> However, everyone began to worry that all of this might not have been enough, as the scoreline was still tied at 5–5. The smell of another potential tiebreaker lingered in the air.

Reviewer: ```suggestion
However, everyone began to worry that all of this might not have been enough, as the scoreline was still tied at 5–5. The smell of another potential tiebreaker lingered in the air.
```

::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357)'s final pick was HR2, one which he had handily won in the previous match. However, despite some narrow misses at the start and at the end of this pick, ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) didn't waver once, reducing his miss count from 73 to 7 and getting tantalisingly close to 99% accuracy, leaving ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357) in the dust and securing himself the first break point of the bracket reset.

> With a 5–6 lead, ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) doubled down and picked HR1 — a pick which where, again, ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357) had won previously. By adding insult to injury on the home stretch, ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) finished with an FC, replicating his opponent's feat from the previous match with much better accuracy to boot and denying his chance at another tiebreaker.

Reviewer: ```suggestion
With a 5–6 lead, ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) doubled down and picked HR1 — a pick which where, again, ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357) had won previously. By adding insult to injury on the home stretch, ::{ flag=JP }:: [Grape_Tea](https://osu.ppy.sh/users/9540073) finished with an FC, replicating his opponent's feat from the previous match with much better accuracy to boot and denying his chance at another tiebreaker.
```

Between ::{ flag=US }:: [mBiscuit](https://osu.ppy.sh/users/17061174) winning on [HD1](https://osu.ppy.sh/beatmapsets/2328983#taiko/4994070), [FM1](https://osu.ppy.sh/beatmapsets/2328977#taiko/4994051), [FM2](https://osu.ppy.sh/beatmapsets/2329063#taiko/4994258), [FM3](https://osu.ppy.sh/beatmapsets/2329068#taiko/4994266), [NM4](https://osu.ppy.sh/beatmapsets/2329039#taiko/4994216), a mix of reading, consistency and gimmick picks, and ::{ flag=JP }:: [supesan](https://osu.ppy.sh/users/14623658) winning on [NM3](https://osu.ppy.sh/beatmapsets/2329033#taiko/4994202), [DT2](https://osu.ppy.sh/beatmapsets/2329049#taiko/4994233), [NM2](https://osu.ppy.sh/beatmapsets/2329034#taiko/4994203), [NM5](https://osu.ppy.sh/beatmapsets/2328994#taiko/4994086), [HR1](https://osu.ppy.sh/beatmapsets/2329036#taiko/4994210), and the [tiebreaker](https://osu.ppy.sh/beatmapsets/2329073#taiko/4994275), the more mechanics-focused picks in this pool, this match made for one of the most interesting contrasts to look at for such a close match.

> This clip right below summarises very well how stressful matches in EGTS are to watch, honestly.

Reviewer 2: the author's writing styke shines through a lot yeah

> This clip right below summarises very well how stressful matches in EGTS are to watch, honestly.

Reviewer 2: ```suggestion
This clip right below summarises very well how stressful matches in EGTS are to watch.
```

    </video>
</div>

Tiebreaker reached. 5–5. No other way out of this than by going through the gruesome gauntlet that is [Atomic Disassembler](https://osu.ppy.sh/beatmapsets/2332724#taiko/5005638). The first half was quite trivial for both players, scores separated by 6x100 by the time the second half started, and somehow, still maintaining the same score difference by the time the slowest SV section ended.

By the time of the song's breakdown section, the misses started coming in. ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027) missed 3 in a row, and ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357) traded them back with 15 in a row. Everything seemed hopeless at this point, even with 4 additional misses from ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027), but then the very last section came up...

There are no words for what ended up happening.

<div align="center" class="osu-md__paragraph">
    <video width="95%" controls>
        <source src="https://assets.ppy.sh/media/news/egts-2025-peaceful-vs-kanten_07-3.mp4" type="video/mp4" preload="none">
    </video>
</div>

> ::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357) won with one of the most decisive ways a tiebreaker could end, by a slim 3,775 points, and advanced to the Semifinals in the winners' bracket.

Reviewer 2: ```suggestion
::{ flag=JP }:: [kanten_07](https://osu.ppy.sh/users/11680357) won in one of the most exciting ways a tiebreaker could end, by a slim 3,775 points, and advanced to the Semifinals in the winners' bracket.
```

> Sadly for him, this would not be the last break point, as ::{ flag=KR }:: [Peaceful](https://osu.ppy.sh/users/165027) ended up retaliating by first winning [HR1](https://osu.ppy.sh/beatmapsets/2336376#taiko/5015688) in dominating fashion (an FC), then tying up the match, and finally, winning [DT2](https://osu.ppy.sh/beatmapsets/2336377#taiko/5015689) in one of the most impressive break points to happen, with just 483 points to decide the whole match.

Author: changed to happen -> imaginable, reads well enough to me

Mappers [Ideal](https://osu.ppy.sh/users/3869519) and [Raphalge](https://osu.ppy.sh/users/3918650) tried to reel this insane song in a bit by toning down the mechanical difficulty compared to the Round of 32, as it instead aims to be played with mods by the players left, but even then, it is a struggle for sure.

### Quarterfinals

[![](/wiki/shared/news/2025-05-06-expert-global-taiko-showdown-2025-concludes/banner-atomic-disassembler.png)](https://osu.ppy.sh/beatmapsets/2332724)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/media/news/egts-2025-atomic-disassembler.mp3">
    </audio>
</div>

[Atomic Disassembler.](https://soundcloud.com/yusuke-7328/atomic-disassembler)

This title summarises the goal behind this song pretty well. The tiebreaker for this year's Quarterfinals by [CS4W](https://soundcloud.com/yusuke-7328) is a Gothic Hardcore song mixed with Hi-tech and Drum & Bass elements, an insane mix of genres to be sure.

> The map by [My Angel Seia](https://osu.ppy.sh/users/17797595) has absolutely nothing angelic about it. The easy parts are when 220 BPM streams are mapped at the normal SV, but whenever you see a single SV slowdown in this map, the only answer is to run, frankly.

Reviewer 2: it's not too bad on occasion but don't mind either way

> The map by [My Angel Seia](https://osu.ppy.sh/users/17797595) has absolutely nothing angelic about it. The easy parts are when 220 BPM streams are mapped at the normal SV, but whenever you see a single SV slowdown in this map, the only answer is to run, frankly.

Author: i think the point is half putting emphasis on it, but it may also be used randomly due to how i speak by default lol
i think it's fine to keep since the honestly above was removed in the last commit iirc? will change the "a single SV slowdown" bc that's too weird even for my taste now that i'm re-reading

With a map by [tasuke912](https://osu.ppy.sh/users/2774767), [uone](https://osu.ppy.sh/users/532171910), and [kei821](https://osu.ppy.sh/users/5846289), who were forced to complete it in a total of three days, it did not disappoint and made for an amazing experience overall.

### Grand Finals

[![](/wiki/shared/news/2025-05-06-expert-global-taiko-showdown-2025-concludes/banner-absolute-disorder.png)](https://osu.ppy.sh/beatmapsets/2343534)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/media/news/egts-2025-absolute-disorder.mp3">
    </audio>
</div>

This is the only non-tiebreaker song to be discussed here, but with good reason to do so.

Our Grand Finals HR2 pick, [Acute Disarray - AbsoluTe disoRdeR](https://soundcloud.com/abs_dsr/absolute-disorder), was an original song released for EGTS 2025, but also a 4-way collaboration with three of the rhythm games we've worked with throughout EGTS.

> [Acute Disarray](https://soundcloud.com/abs_dsr), a unit of [AiSS](https://www.youtube.com/channel/UCySx3jmK0bjwiwix_EoKm4w) participating for [Orzmic](https://twitter.com/orzmic), [rN](https://soundcloud.com/rurinn) participating for [Phigros](https://twitter.com/Phigros_PGS), [takehirotei](https://soundcloud.com/takehirotei) participating for GTS and [Riya](https://soundcloud.com/riyaelectronic) participating for [A Dance of Fire and Ice](https://7thbe.at/), released the song on each respective game.

Author: ok to be fair i think i missed even the original edit for this since it's NOT out on every game yet either way

> [Acute Disarray](https://soundcloud.com/abs_dsr), a unit of [AiSS](https://www.youtube.com/channel/UCySx3jmK0bjwiwix_EoKm4w) participating for [Orzmic](https://twitter.com/orzmic), [rN](https://soundcloud.com/rurinn) participating for [Phigros](https://twitter.com/Phigros_PGS), [takehirotei](https://soundcloud.com/takehirotei) participating for GTS and [Riya](https://soundcloud.com/riyaelectronic) participating for [A Dance of Fire and Ice](https://7thbe.at/), released the song on each respective game.

Reviewer 2: "has now" if so, but it doesn't sound necessary

> [Acute Disarray](https://soundcloud.com/abs_dsr), a unit of [AiSS](https://www.youtube.com/channel/UCySx3jmK0bjwiwix_EoKm4w) participating for [Orzmic](https://twitter.com/orzmic), [rN](https://soundcloud.com/rurinn) participating for [Phigros](https://twitter.com/Phigros_PGS), [takehirotei](https://soundcloud.com/takehirotei) participating for GTS and [Riya](https://soundcloud.com/riyaelectronic) participating for [A Dance of Fire and Ice](https://7thbe.at/), released the song on each respective game.

Author: uhhhhh okay let's call it a wrap

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/6177263" style="background-image: url('/wiki/shared/avatars/Kasumi-sama.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/6177263">Kasumi-sama</a></p>

After your experiences in tournaments, how do you think the tournament scene should move forward with its mappool system?

<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/9864847" style="background-image: url('/wiki/shared/avatars/overdahedge2015.jpg')"></a>

<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/9864847">overdahedge2015</a></p>

I sort of have two answers to this.

As a staff member, I want to say there's a reasonable future where all mappools use only fully licensed music and can still be great, but while it's getting easier every month i think it's still very hard to execute when you don't have the sheer manpower a tourney like EGTS has.

> As someone that's pooling for RGTS right now, it's unreal and honestly, a bit demotivating, the amount of times I have thought "oh this map would be great", then looked in the source field and saw that the song's from another rhythm game, but we're still making it work, with the help of a good but not EGTS-level amount of custom mappers.

Reviewer 3: ```suggestion
As someone that's pooling for RGTS right now, it's unreal, and honestly a bit demotivating, the amount of times I have thought "oh this map would be great", then looked in the source field and saw that the song's from another rhythm game, but we're still making it work, with the help of a good but not EGTS-level amount of custom mappers.
```
maybe taking a lot of liberties with editing my own text here but it's my text :trollface: WHY CAN YOU USE THE TROLLFACE ON GITHUB
